Automattic/wp-calypso
# Publicize: use only one panel

`PluginPrePublishPanel` is a wrapper for `PanelBody` so we can use only the former one directly. Otherwise we get double panel-border.

See [`PluginPrePublishPanel` in Gutenberg](https://github.com/WordPress/gutenberg/blob/3d3c2a4d52903131f68d2694bd551408561fa778/packages/edit-post/src/components/sidebar/plugin-pre-publish-panel/index.js)

#### Changes proposed in this Pull Request

Remove `PanelBody` and move its props to `PluginPrePublishPanel`.

#### Testing instructions

- Build blocks using this branch, e.g. using gutenpack-jn
- Publish a post in Gutenberg

Observe no more double border around the menu item.

##### Before
<img width="319" alt="screen shot 2018-10-31 at 18 00 57" src="https://user-images.githubusercontent.com/87168/47802182-69ef5600-dd38-11e8-8e95-032e6d3148dc.png">


##### After
<img width="298" alt="screen shot 2018-10-31 at 18 10 28" src="https://user-images.githubusercontent.com/87168/47802161-63f97500-dd38-11e8-9ee6-0ce58c11f689.png">
